frontend/src/api/alerts/create.ts

```diff
@@ -7,6 +7,7 @@ const create = async (
 ): Promise<SuccessResponse<PayloadProps> | ErrorResponse> => {
 	const response = await axios.post('/rules', {
 		...props.data,
+		version: 'v4',
```

Comment:
we are defining v4 for all the case, no fallback or prop based, is this expected?

Reply:
@SagarRajput-7, based on the first point in this [comment](https://github.com/SigNoz/engineering-pod/issues/1824#issuecomment-2376986112)  by @srikanthccv

> By default, any new alert created should be v4


we have to send v4 for new alerts.